### ISSUE TITLE
Request for sdv.constraint.ColumnFormula for a wider range of function

### ISSUE THREAD

**Issue by JetfiRex** [sdv-dev/SDV#373]
### Problem Description

Currently I have tried to use the 
```python
some_constraint = sdv.constraints.ColumnFormula(
    column="some_column",
    formula=(some lambda function),
    handling_strategy='reject_sampling'
)
```                        
It does not report an error for now. However, I then tried to have:
```python
model = CopulaGAN(... , constraints = [some_constraint], ...)
model.fit(real)
```
(The lambda function is checked and other attributes with the model is checked to be correct. If I changed the lambda function to equivalent function like:
```python
def function():
    return ...
```
It runs well
And it turns out to have:
```python
AttributeError                            Traceback (most recent call last)
<ipython-input-240-feec9f2fa8ef> in <module>
     30 
     31     # Fit the CopulaGAN
---> 32     model.fit(real)
     33 
     34     # Create 600 rows of data

...\conda\3\envs\py36\lib\site-packages\sdv\tabular\base.py in fit(self, data)
    108                      self._metadata.name, data.shape)
    109         if not self._metadata_fitted:
--> 110             self._metadata.fit(data)
    111 
    112         self._num_rows = len(data)

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\metadata\table.py in fit(self, data)
    488 
    489         LOGGER.info('Fitting constraints for table %s', self.name)
--> 490         constrained = self._fit_transform_constraints(data)
    491         extra_columns = set(constrained.columns) - set(data.columns)
    492 

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\metadata\table.py in _fit_transform_constraints(self, data)
    349                 constraint = constraint.to_dict()
    350 
--> 351             constraint = Constraint.from_dict(constraint)
    352             self._constraints[idx] = constraint
    353 

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\constraints\base.py in from_dict(cls, constraint_dict)
    211                 constraint_class = subclasses[constraint_class]
    212 
--> 213         return constraint_class(**constraint_dict)
    214 
    215     def to_dict(self):

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\constraints\base.py in __init__(self, *args, **kwargs)
     64                 self.__kwargs__.update(dict(zip(arg_names, args)))
     65 
---> 66             old__init__(self, *args, **kwargs)
     67 
     68         __init__.__doc__ = old__init__.__doc__

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\constraints\tabular.py in __init__(self, column, formula, handling_strategy)
    320     def __init__(self, column, formula, handling_strategy='transform'):
    321         self._column = column
--> 322         self._formula = import_object(formula)
    323         super().__init__(handling_strategy)
    324 

...\.julia\conda\3\envs\py36\lib\site-packages\sdv\constraints\base.py in import_object(obj)
     36     if isinstance(obj, str):
     37         package, name = obj.rsplit('.', 1)
---> 38         return getattr(importlib.import_module(package), name)
     39 
     40     return obj

AttributeError: module '__main__' has no attribute '<lambda>'
```

Besides the lambda expression, if the function is a returned function within another function, it will also have this kind of attribution error.

### Expected behavior

It should run well.

**Comment by amontanez24** [comment]
Thank you for reporting this @JetfiRex . I was able to reproduce and will be looking into a potential solution

**Comment by pvk-developer** [comment]
Solved by #378
